src/Pages/Templates/TemplatesTable/TemplatesTable.tsx

```diff
@@ -154,7 +154,8 @@ const TemplatesTable = () => {
     versionDisplay,
   } = useArchVersion();
 
-  const actionTakingPlace = isLoading || isFetching || repositoryParamsLoading || isDeleting;
+  const actionTakingPlace =
+    isLoading || isFetching || repositoryParamsLoading || isDeleting || isFetchingSubscriptions;
```

Comment:
We wouldn't add isFetchingSubscriptions anywhere other than the top level app.tsx

Reply:
ok, resolved [here](https://github.com/content-services/content-sources-frontend/pull/328/commits/92ada247ce4e2cfe19fb4e6adac241f9ae08e54d) 🫡